### PR TITLE
Added delete expiring files functionality

### LINKED ISSUE #6: Prune
Bulk delete files with expiring date passed

### PR DESCRIPTION
**Pull request by NeillGiraldo**
Fixes #6

### CODE REVIEW

File: delete_ms/delete_app.py [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
 def hello_docker():
-    return "Hello DOCKER & Delete"
+    print("Hello DOCKER & Delete")
+
 
 if __name__ == "__main__":
     app.debug = True
     app.config['SEND_FILE_MAX_AGE_DEFAULT'] = 0
     app.run(host=environ["HOST_URL"],port=int(environ["HOST_PORT"]))
+    #app.run(port=8080)
```
Comment (maasencioh) [inline delete_ms/delete_app.py]: Why not remove this?

File: delete_ms/model/File.py [hunk 2]
```diff
@@ -8,6 +8,9 @@
 host = environ["DB_URL"]
 port = environ["DB_PORT"]
 database = environ["DB_NAME"]
+#database = "blinkbox_files"
+#host = "127.0.0.1"
+#port = "5984"
```
Comment (maasencioh) [inline delete_ms/model/File.py]: idem

File: api.raml [hunk 3]
```diff
@@ -7,3 +7,9 @@ mediaType: application/json
     description: Delete file
     delete:
         description: Delete the matching file
+
+/delete_expiring_files/{exp_date}
```
Comment (maasencioh) [inline api.raml]: A couple of things:
  * it's missing a colon at the end of the line
  * usually in an API you use a single base, so I'll do something like

```
/delete:
  /file/{id}:
  /bulk/{exp_date}:
```

**Review by maasencioh — APPROVED** [review]
There're just a couple of nits

File: delete_ms/delete_app.py (hunk 1 above)
Comment (NeillGiraldo) [reply delete_ms/delete_app.py]: Reason: Local testing but decided to remove line
Status: Fixed

File: delete_ms/model/File.py (hunk 2 above)
Comment (NeillGiraldo) [reply delete_ms/model/File.py]: Reason: Local testing but decided to remove lines
Status: Fixed

File: api.raml (hunk 3 above)
Comment (NeillGiraldo) [reply api.raml]: Status: Fixed

**Pull request merged**